packages/core/src/command.ts

```diff
@@ -6,7 +6,7 @@ import {toCommandValue} from './utils'
 // We use any as a valid input type
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-interface CommandProperties {
+export interface CommandProperties {
```

Comment:
do we have to export this? do we intend for customers to use this?

if not, we might want to move into an "internal" module